test/functional/sqlsrv/MsHelper.inc

```diff
@@ -336,18 +389,18 @@ function createTable($conn, $tbname, $columnMetaArr)
     }
     $colDef = rtrim($colDef, ", ");
     $createSql = "CREATE TABLE $tbname ( $colDef )";
-    sqlsrv_query($conn, $createSql);
+    return sqlsrv_query($conn, $createSql);
 }
 
 /**
  * Insert a row into a table
  * @param  object  $conn : sqlsrv connection object
  * @param  string  $tbname : name of the table for the row to be inserted
- * @param  array  $inputs : an associative array column name and its value, which may be a 
+ * @param  array  $inputs : an associative array column name and its value, which may be a
  *                          literal or a BindParamOption object
  * @param  bool  $r : true if the row was successfully inserted, otherwise false. Default value is null to make this parameter optional.
  * $param  string  $api : SQLSRV API used for executing the insert query
- *                      accepted values: INSERT_QUERY, INSERT_PREPARE, INSERT_QUERY_PARAMS, INSERT_PREPARE_PARAMS
+ *                        accepted values: INSERT_QUERY, INSERT_PREPARE, INSERT_QUERY_PARAMS, INSERT_PREPARE_PARAMS
```

Comment:
since you've already defined these constants, they are no longer string, use int

Reply:
I changed it to `int`